kubernetes/deployment.yml

```diff
@@ -15,3 +16,14 @@ spec:
         image: gcr.io/google_containers/kube-state-metrics:v0.4.1
         ports:
         - containerPort: 8080
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8080
+            scheme: HTTP
+          initialDelaySeconds: 8
+          timeoutSeconds: 4
+        resources:
+          limits:
+            cpu: .1
+            memory: 75M
```

Comment:
I just doubled/tripled what it was using in my cluster. Which is ~16nodes with ~200pods running. We can make it 100M to be safe? But it just needs some kind of limit to be safe.